### DISCUSSION TITLE
I am getting Issues in Pull Request

### DISCUSSION THREAD

**Discussion by Shariq2003** [recodehive/awesome-github-profiles#772] [Q&A]
@sanjay-kv sir I facing issues during the creation of PR, 
For my assigned issue, I made changes in only two files by taking a new branch from gh-pages branch -> made changes -> git add and commit then pushing to my feature branch,
However, upon creating a PR I have seen that the PR shows multiple changes along with the multiple commits (of many people) in my PR.
However, I made only two file changes. (My laptop's feature also shows only two file changes.)

![image](https://github.com/user-attachments/assets/3fad88b0-4dea-41d8-924c-96ba731c42b0)

That's why I have closed the PR.

@sanjay-kv please help

**Comment by sanjay-kv** [comment]
@dinxsh to help with

**Reply by Shariq2003** [reply]
I have solved the issue, thanks @sanjay-kv sir
